dev/tools/install-agent-sandbox

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# Copyright 2025 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+import subprocess
+import time
+
+def get_latest_agent_sandbox_version():
+    """Fetches the latest Agent Sandbox version from the GitHub API."""
+    url = "https://api.github.com/repos/kubernetes-sigs/agent-sandbox/releases/latest"
+    response = requests.get(url)
+    response.raise_for_status()
+    tag_name = response.json()["tag_name"]
+    return tag_name.lstrip("v")
+
+def run_command(command):
+    """Runs a shell command and prints its output."""
+    print(f"Running command: {" ".join(command)}")
+    subprocess.run(command, check=True)
+
+def main():
+    """Installs the latest version of Agent Sandbox."""
+    try:
+        version = get_latest_agent_sandbox_version()
+        print(f"Installing agent-sandbox version: {version}")
+ 
+	    # kubectl apply -f https://github.com/kubernetes-sigs/agent-sandbox/releases/download/v0.1.0-rc.0/manifest.yaml
+        install_command = [
+            "kubectl", "apply", "-f",
+            f"https://github.com/kubernetes-sigs/agent-sandbox/releases/download/v{version}/manifest.yaml"
+        ]
+        run_command(install_command)
+
+        print("Waiting for 5 seconds...")
+        time.sleep(5)
+
+        kubectl_get_pods_command = ["kubectl", "get", "pods", "-n", "agent-sandbox-system"]
+        run_command(kubectl_get_pods_command)
+
+        print("Agent Sandbox installed successfully.")
+
+    except requests.exceptions.RequestException as e:
+        print(f"Error fetching Agent Sandbox version: {e}")
+    except subprocess.CalledProcessError as e:
```

Comment:
The generic `except Exception as e:` is too broad. It can catch and hide unexpected errors. It's better to catch more specific exceptions to understand the root cause of failures better.